notebook/static/notebook/js/actions.js

```diff
@@ -437,7 +437,18 @@ define(function(require){
                 $('.header-bar').toggle();
                 events.trigger('resize-header.Page');
             }
-        }
+        },
+        'escape': {
```

Comment:
Ahh, got it. I didn't try it, read the code, but missed the command mode
part. However, I am still not thrilled about using esc for yet another
action. It already does:

* Closes modal dialogs
* Closes the pager
* Enter command mode
* Closes other things like tab completion

Because of its strong usage for closing UI elements and command mode, I
think that even with a scoped keyboard shortcuts we want to avoid using it
for more unrelated things.

On Mon, Oct 26, 2015 at 12:49 PM, Jonathan Frederic <
notifications@github.com> wrote:

> In notebook/static/notebook/js/actions.js
> <https://github.com/jupyter/notebook/pull/657#discussion_r43019194>:
>
> > @@ -437,7 +437,18 @@ define(function(require){
> >                  $('.header-bar').toggle();
> >                  events.trigger('resize-header.Page');
> >              }
> > -        }
> > +        },
> > +        'escape': {
>
> agreed. [image: :heavy_check_mark:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/notebook/pull/657/files#r43019194>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com